Unix/configure

```diff
@@ -1344,11 +1344,50 @@ rm -f $tmpdir/pthread_rwlock_t_func
 
 ##==============================================================================
 ##
+## Check whether SSL 1.0.x is installed on AIX platforms
+##
+##==============================================================================
+
+echo $echon "checking if SSL 1.0.x is installed on AIX platform ... $echoc"
```

Comment:
No. I always echo that on all platforms, but non AIX platforms say "N/A". It was most consistent with everything else in configure (we never had a platform-specific test like this).